Library/Homebrew/test/test_diagnostic.rb

```diff
@@ -141,4 +142,80 @@ def test_check_user_curlrc
         @checks.check_user_curlrc
     end
   end
+
+  def test_check_for_unsupported_curl_vars
+    MacOS.stubs(:version).returns :yosemite
+    ENV["SSL_CERT_DIR"] = "/some/path"
+
+    assert_match "SSL_CERT_DIR support was removed from Apple's curl.",
+      @checks.check_for_unsupported_curl_vars
+  end
+
+  def test_check_for_config_scripts
+    mktmpdir do |path|
+      FileUtils.touch "#{path}/foo-config"
```

Comment:
This needs to be made executable. The reason this doesn't fail is because `check_for_config_scripts` doesn't check if what it finds is executable – which it should.